_maps/configs/pirate_tortuga.json

```diff
@@ -0,0 +1,46 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"map_name": "Tortuga-class Converted Cargo Freighter",
+	"prefix": "ISV",
+	"namelists": [
+		"PIRATES",
+		"BRITISH_NAVY"
+	],
+	"map_short_name": "Tortuga-class",
+	"map_path": "_maps/shuttles/pirate/pirate_tortuga.dmm",
+	"description": "Originally a cargo transport vessel, the Tortuga class is an improvised combat vessel under the ranks of the Frontiersmen fleet. Clever field modifications include the conversion of the cargo bay to a hangar, and additional padding around the ship in the form of asteroid rocks. It serves a dual purpose, both as a capable ship for establishing a military enclave in a sector, and as a sturdy vessel for reconnaissance. The nimble Brawler-class subshuttle of the Tortuga is offset by its very low overall mobility and loss of operation if the subshuttle were to be lost.",
+	"tags": [
+		"Combat",
+		"Subshuttle"
+	],
+	"limit": 1,
+	"job_slots": {
+		"Captain": {
+			"outfit": "/datum/outfit/job/frontiersmen/captain",
+			"officer": true,
+			"slots": 1
+		},
+		"Shipswain": {
+			"outfit": "/datum/outfit/job/frontiersmen/hos",
+			"officer": true,
+			"slots": 1
+		},
+		"Aidman": {
+			"outfit": "/datum/outfit/job/frontiersmen/doctor",
+			"slots": 1
+		},
+		"Sapper": {
```

Comment:
Another awkard job name, given the sapper's primary job is actually going to be maintaining the ship, with occasional assistance for breaches. I'd just go with Mechanic or something similar. As with the doctor slot above, you don't need to use completely faction-unique names for every role, as long as they fit the faction's flavor and suit the job. You also might want to give them a second engineer if you intend for one to join the boarders on the shuttle.